bluesky/protocols.py

```diff
@@ -151,26 +171,20 @@ def collect(self) -> Generator[Dict[str, Any], None, None]:
         """
         ...
 
-    def describe_collect(self) -> Dict[str, Configuration]:
+    # TODO: is this actually right? Do we need collect_asset_docs too?
```

Comment:
I guess `collect_asset_docs` outputs resource and datum documents? I found this:
https://blueskyproject.io/event-model/data-model.html?highlight=path_semantics#resource-document

but no docs on how to write collect_asset_docs anywhere...